helix-front/server/config.example.ts

```diff
@@ -5,20 +5,21 @@ export const HELIX_ENDPOINTS = {
     },
   ],
 };
-
 export const SESSION_STORE = undefined;
-
 export const SSL = {
   port: 0,
   keyfile: '',
   certfile: '',
   passfile: '',
   cafiles: [],
 };
-
 export const LDAP = {
   uri: 'ldap://example.com',
   base: 'DC=example,DC=com',
   principalSuffix: '@example.com',
   adminGroup: 'admin',
 };
+
+export const IDENTITY_TOKEN_SOURCE = undefined; // 'www.example.com';
+export const CUSTOM_IDENTITY_TOKEN_REQUEST_BODY = {};
```

Comment:
Are there specific attributes that are required to be present for  `CUSTOM_IDENTITY_TOKEN_REQUEST_BODY`


Does it make sense to add a `type` or `interface for `CUSTOM_IDENTITY_TOKEN_REQUEST_BODY`
Example
```
export const CUSTOM_IDENTITY_TOKEN_REQUEST_BODY: {some_inline_type} = {};
```

Reply:
Good question.  I don't think so, since by definition it is custom, so it totally depends on the Identity Source / Auth Provider that you decide to use helix-front with.  It would probably make sense to explicitly type this as `any` to communicate that better. Thanks for the suggestion!

Reply:
Added comments to clarify this.